### PR TITLE
Fix misspelled enum

### PR DESCRIPTION
**Pull request by jamill**
Forward not Foward.

### CODE REVIEW

File: LibGit2Sharp/MergeOptions.cs [hunk 1]
```diff
@@ -118,7 +118,7 @@ public enum FastForwardStrategy
         /// <summary>
         /// Do not fast-forward. Always creates a merge commit.
         /// </summary>
-        NoFastFoward = 1, /* GIT_MERGE_NO_FASTFORWARD */
```
Comment (nulltoken) [inline LibGit2Sharp/MergeOptions.cs]: How about not dropping this entry, but decorating it with an [Obsolete("blablblabla..)")] attribute?

**Comment by nulltoken** [comment]
Good catch!

File: LibGit2Sharp/MergeOptions.cs (hunk 1 above)
Comment (jamill) [inline LibGit2Sharp/MergeOptions.cs]: Good idea

**Comment by nulltoken** [comment]
:bomb:

**Pull request merged**